docker-compose.yml

```diff
@@ -1,114 +1,96 @@
-version: '3.7'
-
-volumes:
-  pg_data:
-
+---
 services:
-  solr:
-    image: solr:8-slim
+  app:
+    depends_on:
+      - db
+      - solr
+      - redis
+    image: umich-arclight
+    build:
+      dockerfile: Dockerfile
+      context: .
     ports:
-      - "8983:8983"
+      - "3000:3000" # Rails
+      - "1234:1234" # RubyMine
+      - "26162:26162" # RubyMine
     environment:
-      - SOLR_JAVA_MEM=-Xms1024m -Xmx1024m
-      - SOLR_HEAP=1024m
+      - RAILS_ENV=development
```

Comment:
Could you add the analytics environment settings?

Reply:
Okay, the Google stuff, yes.

Reply:
```
    environment:
      - RAILS_ENV=development
      - FINDING_AID_DATA=/var/opt/app/data
      - GOOGLE_ANALYTICS_DEBUG=false
      - GOOGLE_ANALYTICS_TRACKING_ID=0
      - DATABASE_URL=postgresql://postgres:postgres@db/umich-arclight-development
      - SOLR_URL=http://solr:8983/solr/umich-arclight-development
      - REDIS_URL=redis://redis:6379
```